runtime/core/decoder/torch_asr_decoder.cc

```diff
@@ -37,21 +36,6 @@ TorchAsrDecoder::TorchAsrDecoder(
     searcher_.reset(new CtcWfstBeamSearch(*fst_, opts.ctc_wfst_search_opts));
   }
   ctc_endpointer_->frame_shift_in_ms(frame_shift_in_ms());
-  InitPostProcessing();
-}
-
-void TorchAsrDecoder::InitPostProcessing() {
-  fst::SymbolTableIterator iter(*symbol_table_);
-  std::string space_symbol = kSpaceSymbol;
-  while (!iter.Done()) {
-    if (iter.Symbol().size() > space_symbol.size() &&
-        std::equal(space_symbol.begin(), space_symbol.end(),
-                   iter.Symbol().begin())) {
-      wp_start_with_space_symbol_ = true;
-      break;
-    }
-    iter.Next();
-  }
```

Comment:
wp_start_with_space_symbol_ is no longer needed as we assume that the decoding type must be ctcprefixbeamsearch if symbol_table contains '_'